### DISCUSSION TITLE
Drag a .blend file to open it in Blender via Library Tab

### DISCUSSION THREAD

**Discussion by fram1963** [Victor-IX/Blender-Launcher-V2#68] [General]
Using version 2.0.24 - "Added the possibility to launch a .blend file by dragging and dropping it onto a Blender version in the Library tab."

When I drag my .blend file on the desired Blender version in the Library tab, it does nothing - no launching of Blender at all.  What am I doing wrong - do I have to hold down Alt or Ctrl or Shift while dragging the file?

Any help appreciated.

Edit:  I tested on my PC with an AMD CPU and Vega GPU and it works fine.  It doesn't on my PC with Intel CPU and Nvidia GPU.

**Comment by zeptofine** [comment]
ok, and I will assume it runs fine without specifying a blendfile

What OS are you on? are there any spaces in the file name?

I need to make some debugging statements when we "launch" a build

edit: Does the build highlight in green if you hover a file over it?

**Reply by fram1963** [reply]
It works fine without specifying a blendfile.

I'm running Windows 10 Pro on my Intel PC and Windows 10 Pro on my AMD PC (both up to date)

It does highlight in green on both PC's
 - on my AMD PC when I release the mouse button, it starts Blender opening the file I dragged on.
  - on my Intel PC when I release the mouse button I get the following:
![Blender Launcher Intel PC](https://github.com/Victor-IX/Blender-Launcher-V2/assets/66740425/b06ff291-da83-4420-b906-82531f75a26c)

Notice that the orange circle appears and then nothing happens - it reverts to the default view of the interface.

**Reply by fram1963** [reply]
Any news on this issue?  Still not working on PC in BL 2.1.24.

Also, SHIFT-click on LAUNCH to open last blend file doesn't work on PC in BL 2.1.24.

**Reply by Victor-IX** [reply]
Can you send your .log file?

**Reply by fram1963** [reply]
Yes.  What log do you need?  I'm not sure how to get that - pointers please

**Reply by Victor-IX** [reply]
you can find the log file here on Windows
`%localappdata%\Blender Launcher`

**Comment by fram1963** [comment]
I did two test on my PC.

1- Drag a .blend file on the Launch button (there's a "+Copy" that appears over the button), release and nothing happens - only thing to note the orange circle with 1 that appears for a few seconds then disappear.
2- SHIFT-click LAUNCH button to open previous .blend file - LAUNCH button changes to PREVIOUS, release and nothing happen.

the log file only have one line and I don't think anything goes in the log since the interface doesn't start anything - I have attached it.

I did the same tests on my AMD rig and they both worked fine.

[Blender Launcher.log](https://github.com/Victor-IX/Blender-Launcher-V2/files/14908926/Blender.Launcher.log)

**Reply by Victor-IX** [reply]
I forget to mention you need to run BL in `-debug`, to have all the log
To do that, in a Windows terminal, move where the Blender Launcher.exe is located --> `cd "Folder/Path"`
Run the BL in debug --> `./"Blender Launcher.exe" -debug`

After that try to open the previous .blend file and send the log file

**Comment by fram1963** [comment]
On my PC (Intel), in debug mode, I tried drag-and-drop .blend file and also open previous .blend file.
[Blender Launcher.log](https://github.com/Victor-IX/Blender-Launcher-V2/files/14910149/Blender.Launcher.log)

On my PC (AMD), in debug mode, I tried drag-and-drop .blend file and also open previous .blend file.
[AMD - Blender Launcher.log](https://github.com/Victor-IX/Blender-Launcher-V2/files/14910159/AMD.-.Blender.Launcher.log)

**Reply by Victor-IX** [reply]
and it's not working in both cases?

**Reply by fram1963** [reply]
No.  It only works on my AMD PC.

**Reply by Victor-IX** [reply]
I don't see anything weird in the log file...
What you can try:
 - Double-check if launching blender from the launcher is working 
 - Try opening a small .blend file with drag and drop on a Blender version with default settings (no add-on)
 - Try launching Blender with command line with a specific file `./blender.exe "D:/Graphics/Scenes/Blender/Chipp Ship.blend"`
    - If this doesn't work, the issue doesn't come from the launcher but from Blender
 - Reinstall Blender Launcher manually

**Reply by fram1963** [reply]
•	Double-check if launching blender from the launcher is working
It works.

•	Try opening a small .blend file with drag and drop on a Blender version with default settings (no add-on)
Does not work – nothing happens when drag-and-drop

•	Try launching Blender with command line with a specific file ./blender.exe "D:/Graphics/Scenes/Blender/Chipp Ship.blend"
It works.  Blender opens with the specified .blend file

•	Reinstall Blender Launcher manually
Reinstalled but same result – nothing happens on drag-and-drop or open previous file
